DIPTest/DIPTest/Assembly/ServiceAssembly.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  ServiceAssembly.swift
+//  DIPTest
+//
+//  Created by Uladzislau Abramchuk on 12/6/20.
+//
+
+import Dip
+import UIKit
+
+// MARK: - Login Service
+
+final class ServiceAssembly: Assembly {
+    func assemble(container: DependencyContainer) {
+        container.register(.singleton, type: LoginServiceProtocol.self) { LoginService() }
```

Comment:
Может так будет лучше?)
```suggestion
        container.register(.singleton) {
            LoginService() as LoginServiceProtocol
        }
```

Reply:
определенно)